tests/components/homeassistant_sky_connect/test_config_flow.py

```diff
@@ -7,11 +7,7 @@
 
 from homeassistant.components import homeassistant_sky_connect, usb
 from homeassistant.components.homeassistant_sky_connect.const import DOMAIN
-from homeassistant.components.zha.core.const import (
-    CONF_DEVICE_PATH,
-    DOMAIN as ZHA_DOMAIN,
-    RadioType,
-)
+from homeassistant.components.zha.core.const import DOMAIN as ZHA_DOMAIN, RadioType
```

Comment:
AFAICT `CONF_DEVICE_PATH` is still available in `zha.__init__`. Maybe lets change the import path here to `homeassistant.components.zha` instead. That should work.

IMO that's usually better anyway. Other integrations should only import stuff from the integrations `__init__.py`. Everything else is basically an implementation detail.

Reply:
Before this PR was merged, my PR(https://github.com/home-assistant/core/pull/101634) unfortunately had started a workflow, and then all pytest work in the job [failed](https://github.com/home-assistant/core/actions/runs/6457773660/job/17536702667).

I don't see why other components that are not related to the target PR should be tested in the workflow.

Any way, before https://github.com/home-assistant/core/pull/101712 was merged, my PR(https://github.com/home-assistant/core/pull/101634) was affected by this. I didn’t know how to continue the PR process. 

But it is always in the `Collect information & changes data` state.

Is there anything I can do to make my PR work?